packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * https://www.builder.io/c/docs/custom-components-children
+ * src/components/customTabs.tsx
+ */
+
+import { useState } from 'react';
+import { Blocks, BuilderBlock } from '@builder.io/sdk-react';
+
+type Tab = {
+  tabName: string,
+  children: BuilderBlock[]
+}
+type TabProps = {
+  tabList: Tab[];
+  builderBlock: {
+    id: string | undefined;
+  };
+}
+
+const CustomTabs = (props: TabProps) => {
+  const [activeTab, setActiveTab] = useState(0);
+
+  const selectTab = (tab: number) => {
+    setActiveTab(tab);
+  };
```

Comment:
```suggestion
```

Reply:
Yeah, this is redundant. I have removed and updated this change.